src/styles/getTooltipStyles.js

```diff
@@ -116,7 +116,7 @@ export default function getTooltipStyles(
         tooltipWrapperStyle: {
             position: 'fixed',
             backgroundColor: themeColors.heading,
-            borderRadius: variables.buttonBorderRadius,
+            borderRadius: variables.componentBorderRadiusNormal,
```

Comment:
Can you please make this one `componentBorderRadiusSmall` like it was previously? Thanks!